generate-wasm-examples/generate_wasm_examples-webgpu.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+git init bevy-main
+cd bevy-main
+git remote add origin https://github.com/mockersf/bevy
+git pull --depth=1 origin example-runner
+
+cd bevy-main
```

Comment:
Looks to be redundant. Causing this to be logged:
```
./generate_wasm_examples-webgpu.sh: 8: cd: can't cd to bevy-main
```

```suggestion
```

Note it was still working because bash scripts don't stop at failed `cd` commands unless you tell them to like `cd bevy-main || exit`